### PR TITLE
GDAL 3.0 Coordinate transformation backwards compatibility

### PR DESCRIPTION
**Pull request by terovirtanen**
Fix for issue [GDAL 3.0 Coordinate transformation backwards compatibility is broken #44](https://github.com/City-of-Helsinki/django-munigeo/issues/44)

### CODE REVIEW

File: munigeo/importer/helsinki.py [hunk 1]
```diff
@@ -375,9 +381,14 @@ def make_addr_id(num, num_end, letter):
                     continue
 
             num2 = feat.get('osoitenumero2')
-            if num2 == 0:
+            if num2 is None or num2 == 0:
```
Comment (mhieta) [inline munigeo/importer/helsinki.py:384]: How are these address checks related to Axis change here?

**Review by mhieta — COMMENTED** [review]
Looks good, but how are those address changes related to this GDAL Axis order feature?

File: munigeo/importer/helsinki.py (hunk 1 above)
Comment (terovirtanen) [reply munigeo/importer/helsinki.py:384]: These are not related to Axis change. 
But when I tested geo_import I got None errors for example
```
  File "/usr/local/lib/python3.7/site-packages/munigeo/importer/helsinki.py", line 388, in import_addresses
    letter = feat.get('osoitekirjain').strip()
AttributeError: 'NoneType' object has no attribute 'strip'
```

Comment (mhieta) [reply munigeo/importer/helsinki.py:384]: Ok, understood.

**Review by mhieta — APPROVED** [review]
LGTM 👍 :shipit:
